### DISCUSSION TITLE
PCB Files?

### DISCUSSION THREAD

**Discussion by pennltrex** [AK-Homberger/Seatalk-Autopilot-Remote-Control#16] [General]
Do you have the PCB files? I only see PDF.

**Comment by AK-Homberger** [comment]
Hi,

Currently I can't offer the PCB-Information. I have an new PC and KiCad is not installed. The files are somewhere in a former backup. But you can order the PCB at Aisler (https://aisler.net/andreasbluewater/my-project-repository/arduino-remote-pilot)

Regards,
Andresa
